src/app/views/marketplace/marketplace-compare.html

```diff
@@ -0,0 +1,73 @@
+<div class="dashboard-marketplace-compare">
+  <mno-loader ng-show="vm.isLoading" class="top-buffer-4"></mno-loader>
+  <!-- Header -->
+  <div class="row show-header top-buffer-4">
+    <div class="col-md-12">
+      <a ui-sref="home.marketplace()" class="text-muted">
+        <i class="fa fa-chevron-left"></i> {{ 'mno_enterprise.templates.dashboard.marketplace.show.back' | translate }}
```

Comment:
`<i class="fa fa-chevron-left" translate>mno_enterprise.templates.dashboard.marketplace.show.back</i>` is the preferred syntax for translations that are not interpolating other translations or values.